Daedalus.sh.example

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
I don't like having this file in top-level directory. Can it be moved somewhere?

Reply:
I still don't like having this file in top-level directory. Can it be moved somewhere?

Reply:
I still don't like having this file in top-level directory. Can it be moved somewhere?

@v0d1ch ?

Move it to `scripts/launch/daedalus-example.sh`, for example, and write a comment explaining what this file does.
Also make sure you won't break it (notice that it uses relative paths to the script directory, afaiu).